cg-spec/editors_draft.html

```diff
@@ -2671,12 +2671,36 @@ <h2>IANA Considerations</h2>
 
     <p>In <a href="#concrete-syntaxes"></a>, we have proposed extensions to four existing media types, namely `text/turtle` [[TURTLE]], `application/trig` [[TRIG]], `application/n-triples` [[N-TRIPLES]] and `application/n-quads` [[N-QUADS]]. In <a href="#sparql-star"></a>, we have proposed extensions to three more existing media types, namely `application/sparql-query` [[SPARQL11-QUERY]], `application/sparql-results+json` [[SPARQL11-RESULTS-JSON]], and `application/sparql-result+xml` [[RDF-SPARQL-XMLRES]]. While the <a href="https://www.w3.org/community/rdf-dev/">RDF-DEV Community Group</a> has no authority to officially update these media-types, we hope that a future working group with such authority will consider doing so, and include extensions proposed in this report.</p>
 
-    <p>In the meantime, implementers of RDF-star are faced with a dilemma: use the standard media-type identifiers as if their definition had been already updated (optimistic approach); or make it explicit that they are prone to produce or consume embedded triples (pessimistic approach), perhaps by using a different temporary media-type from the unregistered `x.` tree [[RFC6838]], profile links [[RFC6906]], or custom HTTP headers.</p>
+    <p>In the meantime, implementers of RDF-star are faced with a dilemma: use the standard media-type identifiers as if their definition had been already updated (optimistic approach); or make it explicit that they are prone to produce or consume embedded triples (pessimistic approach), perhaps by using a different temporary media-type from the unregistered `x-` tree [[RFC6838]], profile links [[RFC6906]], or custom HTTP headers.</p>
```

Comment:
Happy to revert the change.

My understanding is that `x-` was an old convention for "experimental, temporary" 
(and now completely not a good idea ([Grandfathered Media Types](https://datatracker.ietf.org/doc/html/rfc6838#appendix-A))

`x.` is space of unregistered named for limited scope types "in private, local environments" which are not temporary, so like more "external" but specifically not "temporary".

Reply:
as you write, `x-` is an old convention; and any temporary media-type will not be registered (!) so by nature it belongs to the "unregistered" namespace, right?
Although of course we know that "temporary-ness" is an illusion in that space.

Reply:
"in private, local environments" says, to me, that the `x.` tree is not appropriate, either -- because we're talking about *global*, *experimental*, and (hopefully) *temporary* (until a long-term media type is determined) deployment.

Reply:
According to [RFC6838](https://datatracker.ietf.org/doc/html/rfc6838#page-20), a provisional registration may be appropriate.

> Accordingly, a provisional registration process is provided to
   support early assignment of media type names in the standards tree.
...
> Provisional registrations MAY be updated or abandoned at any time.
   When the registration is abandoned, the media type is no longer
   registered in any sense; it can subsequently be registered just like
   any other unassigned media type name.

But, there's the theory vs. practice issue of ever encouraging the use of a provisional media type.

Reply:
@gkellogg 
I would definitely not take the risk of registering a media-type that was provisionally registered by someone else and then abandoned, even if the RFC allows me to do so! This feels like seeking trouble...

@TallTed 
"temporary" can be understood as a form a "local", in the temporal dimension... ;-)